### PR TITLE
d_a_obj_sekizo OK

### PR DESCRIPTION
**Pull request by ErraticOocca**
Resolves https://github.com/zeldaret/tp/issues/1026

### CODE REVIEW

File: rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp [hunk 1]
```diff
@@ -4,270 +4,148 @@
...
 /* 80CCDD8C-80CCDDE8 00026C 005C+00 1/0 0/0 0/0 .text            Create__14daObj_Sekizo_cFv */
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-asm int daObj_Sekizo_c::Create() {
-    nofralloc
-#include "asm/rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo/func_80CCDD8C.s"
+int daObj_Sekizo_c::Create() {
+    initBaseMtx();
+    cullMtx = mpModel->getBaseTRMtx();
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp:72]: does fopAcM_SetMtx work here?

File: rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp [hunk 2]
```diff
@@ -4,270 +4,148 @@
...
-asm int daObj_Sekizo_c::Execute(Mtx** i_mtx) {
-    nofralloc
-#include "asm/rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo/Execute__14daObj_Sekizo_cFPPA3_A4_f.s"
+int daObj_Sekizo_c::Execute(Mtx** i_mtx) {
+    if (home.roomNo == dComIfGp_roomControl_getStayNo()) {
+        *i_mtx = &mBgMtx;
+        setBaseMtx();
+        if (field_0x5b2) {
+            if (field_0x5b1) {
+                dComIfG_Bgsp().Release((dBgW_Base*)mpBgW);
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp:93]: casting to base shouldnt be necessary, did you include d_bg_w.h?

File: rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp [hunk 3]
```diff
@@ -4,270 +4,148 @@
...
+                field_0x5b1 = false;
+            }
+        } else if (!field_0x5b1) {
+            dComIfG_Bgsp().Regist((dBgW_Base*)mpBgW, (fopAc_ac_c*)this);
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp:97]: neither cast should be necessary here either

File: rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp (hunk 2 above)
Comment (ErraticOocca) [reply rel/d/a/obj/d_a_obj_sekizo/d_a_obj_sekizo.cpp]: I hadn't - including it allowed me to remove the cast.

**Review by TakaRikka — APPROVED** [review]

**Pull request merged**